### ISSUE TITLE
If Prusa i3 detects firmware issue, no notification in octoprint UI happens

### ISSUE THREAD

**Issue by henry-feldman** [foosel/OctoPrint#3280]
#### What were you doing?

1. have a Prusa without the latest software (in my case lower than 3.8.0 - I think it was 3.7.3? and 
2. run a job created in prusaslice 2.1 which has the version check. 
3.Note the print will stop before starting on the printer without an indication in the octoprint UI)

#### What did you expect to happen?
An alert in octoprint to tell me my job needed input on the printer

#### What happened instead?
nothing

#### Did the same happen when running OctoPrint in safe mode?
didn't try this as it doesn't seem to be a safe mode type issue

#### Version of OctoPrint
Version 1.3.11

#### Operating System running OctoPrint

 MacOS 10.14.6, client: OctoPi 0.17.0: server

#### Printer model & used firmware incl. version

as above Prusa i3 mk3 (well to duplicate don't run with latest firmware obviously), now on 3.8.0

#### Browser and version of browser, operating system running browser

Safari 13.0

#### Link to octoprint.log

Not applicable
#### Link to contents of terminal tab or serial.log

Not applicable

#### Link to contents of Javascript console in the browser

Not applicable
#### Screenshot(s)/video(s) showing the problem:

Again the issue is no UI notification, so screenshot would how normal octoprint...

edit: template

**Comment by GitIssueBot** [comment]
Hi @henry-feldman,

It looks like there is some **information missing** from your bug report that will be needed in order to solve the problem. Read the [Contribution Guidelines](https://github.com/foosel/OctoPrint/blob/master/CONTRIBUTING.md) which will provide you with a template to fill out here so that your bug report is ready to be investigated (I promise I'll go away then too!).

If you did not intend to report a bug but wanted to **request a feature or brain storm** about some kind of development, please take special note of the title format to use as described in the [Contribution Guidelines](https://github.com/foosel/OctoPrint/blob/master/CONTRIBUTING.md).

**Please do not abuse the bug tracker as a support forum** - that can be found at [discourse.octoprint.org](https://discourse.octoprint.org). Go there for any kind of issues with network connectivity, webcam functionality, printer detection or any other kind of such support requests or general questions.

Also **make sure you are at the right place** - this is the bug tracker of the official version of OctoPrint, not the Raspberry Pi image OctoPi nor any unbundled third party OctoPrint plugins or unofficial versions. Make sure too that you have **read through the [Frequently Asked Questions](http://faq.octoprint.org)** and searched the [**existing tickets**](https://github.com/foosel/OctoPrint/search?q=&ref=cmdform&type=Issues) for your problem - try multiple search terms please.

I'm marking this one now as needing some more information. Please understand that if you do not provide that information within the next two weeks (until 2019-10-05 17:20 UTC) I'll close this ticket so it doesn't clutter the bug tracker. This is nothing personal, so please just be considerate and help the maintainers solve this problem quickly by following the guidelines linked above. Remember, the less time the devs have to spend running after information on tickets, the more time they have to actually solve problems and add awesome new features. Thank you!

Best regards,
~ Your friendly GitIssueBot

PS: I'm just an automated script, not a human being, so don't expect any replies from me :) Your ticket is read by humans too, I'm just not one of them.

**Comment by GitIssueBot** [comment]
Since apparently some of the required information is still missing, I'm closing this now, sorry. Feel free to request a reopen of this or create a new issue once you can provide **all** [required information](https://github.com/foosel/OctoPrint/blob/master/CONTRIBUTING.md#how-to-file-a-bug-report).
This is nothing personal. Thank you for your collaboration.
